packages/js/plugins/http/src/util.ts

```diff
@@ -50,7 +50,7 @@ export function toAxiosRequestConfig(request: Request): AxiosRequestConfig {
   }, {});
 
   let config: AxiosRequestConfig = {
-    responseType: request.responseType == "BINARY" ? "arraybuffer" : "text",
+    responseType: request.responseType == ResponseType.BINARY ? "arraybuffer" : "text",
```

Comment:
This won't work as enum from schema will be converted to a string.

Reply:
This is not the case, enums are represented as integers when they're coming from Web3APIs. They're only (optionally) represented as strings when the client is sending an enum as an argument to a Web3API method.